src/Lykke.Cqrs.MessageCancellation/Controllers/MessageCancellationController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Lykke.Cqrs.MessageCancellation.Models;
+using Lykke.Cqrs.MessageCancellation.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Lykke.Cqrs.MessageCancellation.Controllers
+{
+    [PublicAPI, Route("/api/plugin/message-cancellation")]
+    public class MessageCancellationController : Controller
+    {
+        private readonly IMessageCancellationService _messageCancellationService;
+
+        public MessageCancellationController(IMessageCancellationService messageCancellationService)
+        {
+            _messageCancellationService = messageCancellationService;
```

Comment:
Add check for null. This will help to find out the reason why controller is not working in case if somebody forgot to initialize the plugin. This belong to another types which dependes on the services that should be registered in the DI container by application developer (even inderectly, by calling an extension method)